docs/source/getting_started.rst

```diff
@@ -1,2 +1,155 @@
+
+
+*******************************
+Install and Configure Orchestra
+*******************************
+
+Next, let's get Orchestra installed and running. To get the code, just install
+using pip: ``pip install orchestra``.
+
+Orchestra has a number of custom settings that require configuration before
+use. First, download the default orchestra settings file and place it next to
+the project settings file::
+
+  wget https://raw.githubusercontent.com/unlimitedlabs/orchestra/v0.1.0/example_project/example_project/orchestra_settings.py
+  mv orchestra_settings.py your_project/your_project
+
+Next, edit the ``orchestra_settings.py`` file:
+
+* Add ``'simple_workflow'`` to ``settings.INSTALLED_APPS`` in the "General"
+  section if you want to run the demo workflow
+  (:ref:`instructions below <demo-section>`).
+
+* Change settings like the ``ORCHESTRA_PROJECT_API_SECRET`` from ``'CHANGEME'``
+  to more appropriate values.
+
+* Add 3rd party credentials in the "3rd Party Integrations" section so that
+  Orchestra can store files on `Amazon s3 <https://aws.amazon.com/s3/>`_ and use
+  `Google apps <http://apps.google.com>`_ and `Slack <https://slack.com/>`_ to
+  help communicate with expert workers (though these are all optional).
+
+Then, at the bottom of your existing settings file
+(``your_project/your_project/settings.py``), import the orchestra
+settings::
+
+  from .orchestra_settings import setup_orchestra
+  setup_orchestra(__name__)
+
+You'll also need to set up orchestra's URLs, so that Django knows where to
+route users when they view orchestra in the browser. Add the following code
+inside the ``urlpatterns`` variable in
+``your_project/your_project/urls.py``::
+
+    # Registration Views
+    # Eventually these will be auto-registered with the Orchestra URLs, but for
+    # now we need to add them separately.
+    url(r'^orchestra/accounts/',
+        include('registration.backends.default.urls')),
+
+    # Logout then login is not available as a standard django
+    # registration route.
+    url(r'^orchestra/accounts/logout_then_login/$',
+        auth_views.logout_then_login,
+        name='logout_then_login'),
+
+    # Orchestra URLs
+    url(r'^orchestra/',
+        include('orchestra.urls', namespace='orchestra')),
+
+    # Beanstalk Dispatch URLs
+    url(r'^beanstalk_dispatch/',
+        include('beanstalk_dispatch.urls')),
+
+Finally, you'll need to get the database set up. Create your database
+with ``python manage.py migrate``. You'll also want to make sure you have an
+initial worker account set up to try out example workflows. We've provided one
+(username: ``demo``, password: ``demo``) as a fixture which you can load with
+``python manage.py loaddata demo_worker``.
+
+Now Orchestra should be ready to go! If you're confused about any of the above,
+check out our barebones `example project <https://github.com/unlimitedlabs/orchestra/tree/v0.1.0/example_project>`_.
```

Comment:
We should keep a `stable` tag and link to that, that way we don't have to change the links for each version.